utils/config_generator.py

```diff
@@ -0,0 +1,383 @@
+import argparse
+import re
+import subprocess
+import sys
+from io import StringIO
+from pathlib import Path
+
+import ruamel.yaml
+
+from utils.find_corpus import (
+    fetch_mtdata,
+    fetch_news_crawl,
+    fetch_opus,
+    fetch_sacrebleu,
+    get_remote_file_size,
+)
+
+"""
+Generate a training config for a language pair based on the latest production
+training config, taskcluster/configs/config.prod.yml.
+"""
+
+root_dir = Path(__file__).parent.parent
+prod_config_path = root_dir / "taskcluster/configs/config.prod.yml"
+
+pretrained_student_models = {
+    ("ru", "en"): "https://storage.googleapis.com/releng-translations-dev/models/ru-en/better-teacher/student"
```

Comment:
This is a great idea btw! We just need to prepare those models (copy vocab etc)

Reply:
@eu9ene should ask @bhearsum where to find recently trained models

Reply:
If you're talking about the ones I uploaded by hand, these are in https://console.cloud.google.com/storage/browser/moz-fx-translations-data--303e-prod-translations-data;tab=objects?forceOnBucketsSortingFiltering=true&project=moz-fx-translations-data--303e&prefix=&forceOnObjectsSortingFiltering=false.

Anything after I did those uploads only exists in Taskcluster artifacts.

Reply:
ok, so the most recent ones are in the tasks and slightly less recent are in `https://console.cloud.google.com/storage/browser/moz-fx-translations-data--303e-prod-translations-data/models/en-hu/`. I see that the model like https://console.cloud.google.com/storage/browser/moz-fx-translations-data--303e-prod-translations-data/models/en-hu/baseline_enhu_aY25-4fXTcuJNuMcWXUYtQ/student-finetuned?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&project=moz-fx-translations-data--303e needs a vocab copied to the folder to use for training continuation as we started copying it in the pipeline only recently.

Reply:
Your GCP accounts has credentials to do this, FYI. That's going to the be quickest way to unblock this.